assets/js/modules/search-console/settings/settings-view.js

```diff
@@ -19,27 +19,27 @@
 /**
  * WordPress dependencies
  */
-import { Component } from '@wordpress/element';
 import { __ } from '@wordpress/i18n';
-const { siteURL } = global.googlesitekit.admin;
 
-class SearchConsoleSettingStatus extends Component {
-	render() {
-		return (
-			<div className="
-				mdc-layout-grid__cell
-				mdc-layout-grid__cell--span-12
-			">
+/**
+ * Internal dependencies
+ */
+import Data from 'googlesitekit-data';
+import { STORE_NAME } from '../datastore/constants';
+
+export default function SettingsView() {
+	const { useSelect } = Data;
```

Comment:
This would be better to move right below the imports like in other datastore-using components, so that it is not unnecessarily run on every render.

Reply:
Completed